samza-rest/src/main/java/org/apache/samza/monitor/MonitorConfig.java

```diff
@@ -96,4 +98,11 @@ public String getMonitorFactoryClass() {
   public int getSchedulingIntervalInMs() {
     return getInt(CONFIG_SCHEDULING_INTERVAL, DEFAULT_SCHEDULING_INTERVAL_IN_MS);
   }
+
+  public double getSchedulingJitterPercent() {
+    if (containsKey(CONFIG_SCHEDULING_JITTER_PERCENT)) {
+      return getDouble(CONFIG_SCHEDULING_JITTER_PERCENT) / 100.0;
```

Comment:
Config is already a percent, if you divide by 100 and return that's not the jitter percent anymore. Let's do the division at the call site.

Reply:
Fixed.